packages/react-table/src/components/Table/SortColumn.tsx

```diff
@@ -7,7 +7,7 @@ import styles from '@patternfly/react-styles/css/components/Table/table';
 import { SortByDirection } from './Table';
 import { TableText } from './TableText';
 
-export interface SortColumnProps extends React.HTMLAttributes<HTMLButtonElement> {
+export interface SortColumnProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
What does this change do exactly? I think the old type is correct for `<button {...props}>`

Reply:
React.HTMLAttributes is a common subset of all html element attributes, therefore it does not include button specific ones. For better explanation check: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36505#issuecomment-549394273

Reply:
Wonderful, thanks for the explanation!